go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/cert-manager/cert-manager
 
-go 1.23.0
-
-toolchain go1.23.7
+go 1.23.8
```

Comment:
The `toolchain` line was replaced with a `go` line because I ran:

```
 _bin/tools/go get go@1.23.8
```

instead of 

```
 _bin/tools/go get toolchain@1.23.8
```